### PR TITLE
Issue #2 || Created the folder structure and fixed some minor details (not code related)

### LINKED ISSUE #2: Create Folder Structure
Folder structure needs to be as follow:

```bash
src
--components
-----Button.js
-----TextField.js
--container
-----HomePage.js
-----SignInPage.js
-----SignUpPage.js
--images
-----logo.jpg
```

(use empty files inside directories for this issue)

### PR DESCRIPTION
**Pull request by Niaz-Ul-Haque**
I added the folder structure but also did some whitespace removal (as you can see from the code). Everything is the same just added the folder structure.

### CODE REVIEW

**Review by Niaz-Ul-Haque — COMMENTED** [review]
Just cleaned up white spaces and removed/fixed whitespaces throughout entire program.

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,24 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import logo from "./logo.svg";
```
Comment (kelumsampath) [inline src/App.js]: you have changed this logo.svg file location in this pull request (src/logo.svg → src/images/logo.svg). so, it has to update the path here also. Due to that reason build error happed. could you change this path and push again. 
Capture

Comment (Niaz-Ul-Haque) [reply src/App.js]: OHHHHHHHHHHH, I see. That was my bad. I'll fix it right away and do a  PR. Thank you!

**Comment by Niaz-Ul-Haque** [comment]
Done, please do check. tytyty

**Review by kelumsampath — APPROVED** [review]

**Pull request merged**